_layouts/base.html

```diff
@@ -19,6 +19,7 @@
 <!DOCTYPE html>
 <html lang="en">
   <!-- Beautiful Jekyll | MIT license | Copyright Dean Attali 2016 -->
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300;500;600;700;900&family=Work+Sans:wght@300;400;700&display=swap" rel="stylesheet">
```

Comment:
Shouldn't this go into `head.html`?

Reply:
I have no idea. Both seems to work so I moved it to head.html!